cmd/configtxgen/main_test.go

```diff
@@ -59,7 +60,7 @@ func TestMissingOrdererSection(t *testing.T) {
 	config := genesisconfig.Load(genesisconfig.SampleInsecureSoloProfile, configtest.GetDevConfigDir())
 	config.Orderer = nil
 
-	assert.Error(t, doOutputBlock(config, "foo", blockDest), "Missing orderer section")
+	assert.EqualError(t, doOutputBlock(config, "foo", blockDest), "refusing to generate block which is missing orderer section")
```

Comment:
Interestingly enough we follow the convention here, further demonstrating how inconsistent error handling is in Fabric